mm2src/lp_ordermatch/best_orders.rs

```diff
@@ -106,6 +106,10 @@ pub async fn process_best_orders_p2p_request(
 
 pub async fn best_orders_rpc(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>, String> {
     let req: BestOrdersRequest = try_s!(json::from_value(req));
+    let req_coin_conf = coin_conf(&ctx, &req.coin);
```

Comment:
To reduce code duplication a bit could you please add the following
```rust
fn is_wallet_only_ticker(ctx: &MmArc, ticker: &str) -> bool
fn is_wallet_only_conf(conf: &Json) -> bool
```
and replace the code with them wherever possible?